tests/cupyx_tests/scipy_tests/sparse_tests/test_index.py

```diff
@@ -0,0 +1,110 @@
+import unittest
+
+import cupy
+from cupy import sparse
+from cupy import testing
+
+import numpy
+
+import pytest
+
+
+@testing.parameterize(*testing.product({
+    'format': ['csr', 'csc'],
+    'density': [0.1, 0.4, 0.9],
+    'dtype': ['float32', 'float64', 'complex64', 'complex128'],
+    'n_rows': [25, 150],
+    'n_cols': [25, 150]
+}))
+@testing.with_requires('scipy')
+@testing.gpu
+class TestIndexing(unittest.TestCase):
+
+    def _run(self, maj, min=None):
```

Comment:
...and loop over all dtypes here instead:
```suggestion
    @testing.for_dtypes('fdFD')
    def _run(self, maj, min=None, dtype):
```